public/app/features/explore/RichHistory/RichHistoryCard.tsx

```diff
@@ -117,7 +117,7 @@ export function RichHistoryCard(props: Props) {
       <div className={styles.queryCardLeft} onClick={() => onChangeQuery(query)}>
         {query.queries.map((q, i) => {
           return (
-            <div key={`${q}-${i}`} className={styles.queryRow}>
+            <div title="query" key={`${q}-${i}`} className={styles.queryRow}>
```

Comment:
Do we really want to use a title there, instead of `aria-label`?

Reply:
I have removed it as it wasn't used for testing purposes and it is redundant.